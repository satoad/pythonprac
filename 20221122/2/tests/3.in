Ââåñòè ñî ñòàíäàðòíîãî ââîäà è âûâåñòè íà ñòàíäàðòíûé âûâîä áèíàðíûå äàííûå, ïåðâûé áàéò êîòîðûõ \x97 ýòî êîëè÷åñòâî ðàâíûõ (±1 áàéò) ÷àñòåé, íà êîòîðûå íóæíî ðàçäåëèòü îñòàëüíîé ââîä, îòñîðòèðîâàòü ýòè ÷àñòè ìåæäó ñîáîé ïî âîçðàñòàíèþ, è âûâåñòè íà ñòàíäàðòíûé âûâîä (âìåñòå ñî ñòàðòîâûì áàéòîì âíà÷àëå).
