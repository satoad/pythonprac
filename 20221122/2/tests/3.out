Ввести со стандартного ввода и вывести на стандартный вывод бинарные данные, первый байт которых \x97 это количество равных (±1 байт) частей, на которые нужно разделить остальной ввод, отсортировать эти части между собой по возрастанию, и вывести на стандартный вывод (вместе со стартовым байтом вначале).
